src/index/repositories.props

```diff
@@ -50,6 +50,15 @@
         $(VsDevCmd) &amp; build -hostType Core -skipTests -bootstrapOnly
       </PrepareCommand>
     </Repository>
+    <Repository Include="machinelearning">
+      <Url>https://github.com/dotnet/machinelearning</Url>
+      <Projects>
+        src/Microsoft.*/**/*.csproj;
```

Comment:
This can just be `src/**/*.csproj`. No reason to exclude something has a different prefix. Everything under `src` should be indexed.